src/MusicStore/Components/AnnouncementComponent.cs

```diff
@@ -10,18 +10,23 @@ namespace MusicStore.Components
     [ViewComponent(Name = "Announcement")]
     public class AnnouncementComponent : ViewComponent
     {
-        private readonly MusicStoreContext _dbContext;
-        private readonly IMemoryCache _cache;
+        [Activate]
+        public MusicStoreContext DbContext
+        {
+            get;
+            set;
+        }
 
-        public AnnouncementComponent(MusicStoreContext dbContext, IMemoryCache memoryCache)
+        [Activate]
+        public IMemoryCache MemoryCache
```

Comment:
@Praburaj 
I think MemoryCache is not bad name because it is of IMemoryCache..I think we can change the name when we have to change its type to something else. What do you think?